.gitignore

```diff
@@ -34,3 +37,8 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+whappu-9f17f78c8dc4.json
+functions/node_modules/**
+firebase.json
+.firebaserc
+package-lock.json
```

Comment:
Ignoring `package-lock.json` kind of beats the whole point of version locking. NPM 5 (Node 7 and above) added this feature to ensure more consistent versioning for projects

Reply:
I was going to mention this as well, but noticed that the repo also has yarn.lock.
BUT I just noticed that package.json defines npm5 as the npm version to use 🤔

Reply:
True, I just wrote whatever came to mind starting from the top, so I didn't notice the yarn lockfile at the bottom of the diff